tools/sense_studio/templates/project_details.html

```diff
@@ -12,10 +12,18 @@
         <h3 class="ui header center aligned">Settings</h3>
         <div class="ui toggle checkbox hashoverpopup"
            data-content="Should GPU be used for computing features?"
-           onclick="toggleGPU('{{ path }}');">
+           onclick="toggleGPU('{{ path }}', 'use_gpu');">
             <input type="checkbox" id="gpuInput" {% if config.use_gpu %} checked="checked" {% endif %}>
             <label>Use GPU</label>
         </div>
+        <br/>
+        <br/>
+        <div class="ui toggle checkbox hashoverpopup"
+           data-content="Do you want to make project temporal?"
+           onclick="toggleMakeProjectTemporal('{{ path }}', '{{ project }}', 'project_temporal');">
+            <input type="checkbox" id="makeProjectTemporal" {% if config.project_temporal %} checked="checked" {% endif %}>
+            <label>Make Project Temporal</label>
```

Comment:
Instead of a long string "Make Project Temporal", can we rename it to "Temporal Training"?

Reply:
we can let's think of a name which is better understood I think.